readthedocs/core/unresolver.py

```diff
@@ -53,6 +59,24 @@ class UnresolvedURL:
     external: bool = False
 
 
+class OriginType(Enum):
```

Comment:
Not sure about this name, suggestions are welcome.

Reply:
```suggestion
class DomainSourceType(Enum):
```

Perhaps?

Reply:
sold!